__init__.py

```diff
@@ -476,6 +472,85 @@ def get_best_playlist(self, playlist):
         else:
             return None
 
+    def play_song(self, message):
+        """
+        When the user wants to hear a song, optionally with artist and/or album information attached
+        play the song <song>
+        play the song <song> by <artist>
+        play the song <song> off <album>
+        play <song> by <artist> off the album <album>
+        etc.
+        :param message: the utterance as interpreted by Padatious
+        """
+        song = message.data.get('track')
+        artist = message.data.get('artist')
+        album = message.data.get('album')
+        # workaround for Padatious training:
+        if song and not album:
+            m = re.match(r'^play (some |a )?(something|music|track)( by ([\w\s]+?))?$', message.data['utterance'], re.M|re.I)
+            if m:
+                LOG.info("I'm in the play_song handler but I think I'm actually being asked to play something indeterminate. Switching handlers.")
+                self.play_something(message)
+                return
+
+        query = song
+        LOG.info("I've been asked to play a particular song.")
+        LOG.info("\tI think the song is: " + song)
+        if artist:
+            query += " artist:" + artist
```

Comment:
We should try to keep consistent and use `'` instead of `"` for single line strings